packages/just-task/src/logger.ts

```diff
@@ -1 +1,85 @@
-export * from 'just-task-logger';
+import chalk = require('chalk');
```

Comment:
This is NOT new code, just a move/merge of the files `just-task-logger/src/logger.ts` and `just-task-logger/src/mark.ts` (the mark file was small, so it was easier to just consolidate them)